[67645] CTA on Project settings file storage page are too close to another

HDinger opened the pull request:
# Ticket
https://community.openproject.org/wp/67645

# What are you trying to accomplish?
Add space between buttons and swap position

## Screenshots
<img width="804" height="320" alt="Bildschirmfoto 2025-10-01 um 10 53 35" src="https://github.com/user-attachments/assets/a7d945cb-4e01-401a-8446-0e91badf6330" />

HDinger committed:
Add space between buttons

myabc commented:
Why are we showing an `x` icon for the Cancel button here but nowhere else in the application?

myabc reviewed (commented):
Code-wise this looks fine, but I'm not sure about the design: the button ordering follows that of modal dialogs, not other full page forms.

HDinger commented:
> Why are we showing an x icon for the Cancel button here but nowhere else in the application?

We do have that in many places.. It is unfortunately not consistent.

> Code-wise this looks fine, but I'm not sure about the design: the button ordering follows that of modal dialogs, not other full page forms.

For now, I switched the positions back to how they were before and just fixed the spacing. I added a point to the next agenda to discuss that in more general.
